### ISSUE TITLE
System Light  mode and specific app tune to dark doesn't work

### ISSUE THREAD

**Issue by havewisdom** [zenangst/Gray#120]
In the system dark mode, other apps are set to the light mode.
It's not the opposite.
System in light mode Other apps in dark mode do not work.

**Comment by zenangst** [comment]
Hey @havewisdom,

Gray only works if your system is set to use `Dark appearance`. This is a limitation of the OS, not something that Gray can fix I'm afraid.

**Comment by kylechadha** [comment]
Yeah, seeing this on 14.5 as well, I basically hacked around the limitation by setting system to dark and then making everything light but the one app I wanted dark. 

We'll see how annoying that is come nighttime 😂
